### PR TITLE
Contracts v2

### PR DESCRIPTION
**Pull request by ksridharbabuus**
Platform Contracts Upgrade and updated the code as per the Solidity Compiler needs 0.6.2+.
Has a support for the upcoming AGI-2 contracts.

### CODE REVIEW

File: contracts/Registry.sol [hunk 1]
```diff
@@ -204,14 +213,15 @@ contract Registry is IRegistry, ERC165 {
             }
 
             // shorten keys array
-            orgsById[orgId].memberKeys.length--;
+            //orgsById[orgId].memberKeys.length--;
```
Comment (astroseger) [inline contracts/Registry.sol:216]: It is probably cleaner to remove old commented code completely (there are several "length--" comments here)

File: contracts/IRegistry.sol [hunk 2]
```diff
@@ -26,7 +26,7 @@ interface IRegistry {
       * @param orgMetadataURI  MetadataURI of organization to create, must be unique registry-wide.
       * @param members  Array of member addresses to seed the organization with.
       */
-    function createOrganization(bytes32 orgId, bytes orgMetadataURI, address[] members) external;
+    function createOrganization(bytes32 orgId, bytes calldata orgMetadataURI, address[] calldata members) external;
```
Comment (astroseger) [inline contracts/IRegistry.sol:29]: Is calldata cheaper then memory?

File: contracts/Registry.sol (hunk 1 above)
Comment (ksridharbabuus) [reply contracts/Registry.sol]: Thanks, kept it for internal reference. Removed the commented code.

File: contracts/IRegistry.sol (hunk 2 above)
Comment (ksridharbabuus) [reply contracts/IRegistry.sol:29]: calldata is similar to memory.  For all the external methods it is mandatory to keep it as calldata than memory.

**Review by raamb — APPROVED** [review]

**Review by anandrgitnirman — APPROVED** [review]
keeping in mind we need to expedite this, all details already tested in ropsted

**Pull request merged**